<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyEditorAmbCardDialog</class>
 <widget class="QDialog" name="PropertyEditorAmbCardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTextEdit" name="edtPropertyText"/>
     <widget class="QWidget" name="">
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,5">
       <item row="0" column="0">
        <widget class="QGroupBox" name="gbFilters">
         <property name="title">
          <string>Фильтр</string>
         </property>
         <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0">
          <property name="margin">
           <number>4</number>
          </property>
          <property name="spacing">
           <number>4</number>
          </property>
          <item row="0" column="4">
           <widget class="CActionTypeComboBox" name="cmbGroup">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Тип (группа?)</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="CDateEdit" name="edtBegDate">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="CActionStatusComboBox" name="cmbStatus">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="6">
           <widget class="QCheckBox" name="chkHasProperties">
            <property name="text">
             <string>Имеет свойства</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2">
           <widget class="QLabel" name="lblBegDate">
            <property name="text">
             <string>Начат с</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5" colspan="2">
           <widget class="COrgStructureComboBox" name="cmbOrgStructure">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="2">
           <widget class="QLabel" name="lblEndDate">
            <property name="text">
             <string>По</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QCheckBox" name="chkHasAttachedFiles">
            <property name="text">
             <string>Имеет прикрепленные файлы</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="4">
           <widget class="CRBComboBox" name="cmbSpeciality">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Специальность</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="CDateEdit" name="edtEndDate">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="8" rowspan="2">
           <widget class="QDialogButtonBox" name="btnFiltersButtonBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="standardButtons">
             <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
            </property>
           </widget>
          </item>
          <item row="2" column="7">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="CRegistryActionsTableView" name="tblActions"/>
         <widget class="CActionPropertiesTableView" name="tblActionProperties"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CActionStatusComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">Events/ActionStatus.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CActionTypeComboBox</class>
   <extends>QComboBox</extends>
   <header>Events/ActionTypeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CActionPropertiesTableView</class>
   <extends>QTableView</extends>
   <header location="global">Events/ActionPropertiesTable.h</header>
  </customwidget>
  <customwidget>
   <class>CRegistryActionsTableView</class>
   <extends>QTableView</extends>
   <header location="global">Registry/RegistryTable.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PropertyEditorAmbCardDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PropertyEditorAmbCardDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
